buildSrc/src/main/groovy/com/uber/okbuck/composer/AndroidBinaryRuleComposer.groovy

```diff
@@ -23,9 +23,19 @@ final class AndroidBinaryRuleComposer extends AndroidBuckRuleComposer {
             CPU_FILTER_MAP.get(cpuFilter)
         }.findAll { String cpuFilter -> cpuFilter != null }
 
+        Set<String> transformRules = TrasformDependencyWriterRuleComposer.getTransformRules(target)
```

Comment:
Seems like `TransformUtils` is a better name than `TrasformDependencyWriterRuleComposer`